lib/cocoapods_packager.rb

```diff
@@ -1,5 +1,5 @@
 module Pod
   module Packager
-    VERSION = '1.5.0'.freeze
+    VERSION = '1.5.2'.freeze
```

Comment:
would like to keep this PR to just the functionality changes, and a CocoaPods developer can handle bumping and releasing versions